jbx-protocol/juice-interface
# Resolve `/settings/nfts` and `/settings/cycle` conflicting `reconfigureFundingCyclesOf` tx

Very edge case: if someone calls `reconfigureFundingCyclesOf` from `/settings/nfts` in the same cycle as they call it from `settings/cycle` or vice-versa, the prior will be overriden and not applied. See more:
https://discord.com/channels/939317843059679252/1080714793595441182/1151499620707479582

**To test:**

1. Create project without nfts, with a funding cycle of 10-15min (however quick you think you can do steps 2 and 3) and no edit deadline
2. execute an edit cycle tx 
3. launch new nft collection 
4. Wait till next cycle starts, should see all the edit cycle tx changes, plus a new nft collection
5. Repeat steps 1-4 but switch steps 2 and 3
